### PR TITLE
Fix show back button when clicking on tagged user

### PR DESCRIPTION
**Pull request by Tyoda**
Previously a non-working menu button would show in the top left corner when going to a user profile through a user tag (e.g. @Tyoda@lemm.ee). Now a working go back button is shown instead.

### CODE REVIEW

File: app/src/main/java/com/jerboa/MainActivity.kt [hunk 1]
```diff
@@ -362,6 +362,7 @@ class MainActivity : AppCompatActivity() {
                             blurNSFW = appSettings.blurNSFW.toEnum(),
                             showPostLinkPreviews = appSettings.showPostLinkPreviews,
                             drawerState = drawerState,
+                            onBack = appState::popBackStack,
```
Comment (MV-GH) [inline app/src/main/java/com/jerboa/MainActivity.kt:365]: Thanks, but can you remove the default `null` from it. We have noticed these defaults have been footguns. They caused issues like this.

**Review by MV-GH — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/jerboa/MainActivity.kt (hunk 1 above)
Comment (Tyoda) [reply app/src/main/java/com/jerboa/MainActivity.kt:365]: Not sure what you mean by that, sorry. There is no `null` involved.

Comment (Tyoda) [reply app/src/main/java/com/jerboa/MainActivity.kt:365]: Unless you mean from the constructor of PersonProfileScreen, but I didn't write that. Could remove it of course.

Comment (MV-GH) [reply app/src/main/java/com/jerboa/MainActivity.kt:365]: Yes, thats what I mean

Comment (Tyoda) [reply app/src/main/java/com/jerboa/MainActivity.kt:366]: Removed it from both `onBack` and `padding`, and made them null where they were that by default before.

**Review by dessalines — APPROVED** [review]

**Pull request merged**